src/controllers/bestPractices.controller.js

```diff
@@ -3,7 +3,7 @@ import BestPractices from '../models/bestPractices';
 const BestPractice = BestPractices;
 
 export default function createBestPractice(req, res, next) {
-  const agroTest = new BestPractice({ ...req.body });
+  const agroTest = new BestPractice({ ...req.params });
```

Comment:
Are you passing the input data as query parameters?

Reply:
Yes. I'm assuming the data will come from a form.